stable/yugaware/templates/kube-state-metrics-deployment.yaml

```diff
@@ -0,0 +1,66 @@
+{{- if and .Values.kubeStateMetric.install (not .Values.kubeStateMetric.customEndpoint) -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-ksm
+  {{- if .Values.yugaware.service.annotations }}
+  annotations:
+{{ toYaml .Values.yugaware.service.annotations | indent 4 }}
+  {{- end }}
+  labels:
+    app: {{ .Release.Name }}-yugaware-ksm
+    chart: {{ template "yugaware.chart" . }}
+    release: {{ .Release.Name }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ .Release.Name }}-yugaware-ksm
+  replicas: {{ .Values.kubeStateMetric.replicas }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Release.Name }}-yugaware-ksm
+      annotations:
+{{- if .Values.yugaware.pod.annotations }}
+{{ toYaml .Values.yugaware.pod.annotations | indent 8 }}
+{{- end }}
```

Comment:
Same here. These are yugaware specific annotations, so we should remove this I guess?